Manito/Manito/Screens/Login/ViewModel/LoginViewModel.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  LoginViewModel.swift
+//  Manito
+//
+//  Created by COBY_PRO on 10/24/23.
+//
+
+import AuthenticationServices
+import Combine
+import Foundation
+
+final class LoginViewModel: NSObject, BaseViewModelType {
+    
+    // MARK: - property
+    
+    private let loginService: LoginService
```

Comment:
p1;

추상적으로 선언 부탁드립니다!

```suggestion
    private let loginService: LoginSevicable
```

Reply:
[20ae10e](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/580/commits/20ae10e547832dc30a44171c6796afb132fa2a7d)
수정하였습니다.